tests/test_client.py

```diff
@@ -1,5 +1,25 @@
 from replicate.client import Client
 
+import responses
+from responses import matchers
 
-def test_client():
-    client = Client()
+package_version = None
+exec(open('replicate/package_version.py').read())
```

Comment:
You can also just import it here! It's only in `setup.py`. This will actually break the tests -- `CWD` is not necessarily the root directory.

Minor thing though doesn't really matter. It was just the main package where it really mattered -- you don't really want to be spinning up a whole separate python interpreter each time you import...